src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -122,6 +123,18 @@ class ReportActionItemMessageEdit extends React.Component {
         this.debouncedSaveDraft.cancel();
 
         const trimmedNewDraft = this.state.draft.trim();
+
+        // When user tries to save the empty message, it will delete it. Prompt the user to confirm deleting.
+        if (!trimmedNewDraft) {
+            ReportActionContextMenu.showDeleteModal(
+                this.props.reportID,
+                this.props.action,
+                false,
```

Comment:
This should have been `true`. Later this caused minor focus issue - [Composer doesn't get focused automatically after delete message in edit mode](https://github.com/Expensify/App/issues/30135)